audformat/core/utils.py

```diff
@@ -489,8 +489,6 @@ def to_segmented_index(
     """
     if index_type(obj) == define.IndexType.SEGMENTED:
         return obj
-    elif obj.empty:
-        return segmented_index()
```

Comment:
You just added this a few days ago in https://github.com/audeering/audformat/pull/44,
was there no reasoning behind it?

Reply:
The fix was not correct as it only covered the case where `obj` is an `Index`, but `obj` can also be `pd.Series` or `pd.DataFrame` and in that case we don't want to return an index.

Reply:
You said it covered the case where `obj` is an empty `Index`. How is this case now covered? or was the fix not neccessary before?

Reply:
Before we had:

```python
if obj.empty or index_type(obj) == define.IndexType.SEGMENTED:
             return obj
```

and there we had the problem that it would return an `filewise` index if the input was an object with a filewise index.

Reply:
So basically I fixec a bug and introduced another bug. But let's blame the reviewer for not spotting  :)